OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Change Date on Intangible Time Entry

# Description
This PR enables the user with the Change Date on Intangible Time Entry permission to be able to change the date on the Intangible Time Entry form.

## Related PRS (if any):
This frontend PR is related to the [#726](https://github.com/OneCommunityGlobal/HGNRest/pull/726) backend PR.

…

## Main changes explained:
- Added conditional logic so that the user with the permission can access and change the date on the Intangible Time Entry form.

## How to test (please watch the video):
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log in with an owner account
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a user without **Change Time Entry Date** permission under Timelog Management, Edit Time Entries -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to access and change date on another user's dashboard for time log entries.

## Screenshots or videos of changes:




https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/b51419fe-ae6d-430d-bbb8-3807fdbcdf39




## Note:
**Unmute the video**
